### PR TITLE
cmd/tailscale, ipn/ipnlocal: add "peerapi" ping type

### PR DESCRIPTION
**Pull request by bradfitz**
For debugging when stuff like #4750 isn't working.

### CODE REVIEW

**Review by DentonGentry — APPROVED** [review]

File: ipn/ipnlocal/local.go [hunk 1]
```diff
@@ -1715,6 +1736,51 @@ func (b *LocalBackend) Ping(ctx context.Context, ip netaddr.IP, pingType tailcfg
 	}
 }
 
+func peerForIP(nm *netmap.NetworkMap, ip netaddr.IP) (_ *tailcfg.Node, ok bool) {
```
Comment (maisem) [inline ipn/ipnlocal/local.go:1739]: We should at some point just have a `map[netaddr.IP]*tailcfg.Node`

Comment (maisem) [reply ipn/ipnlocal/local.go:1739]: Maybe on the netmap struct

**Review by maisem — APPROVED** [review]

File: ipn/ipnlocal/local.go (hunk 1 above)
Comment (bradfitz) [reply ipn/ipnlocal/local.go]: done

Comment (bradfitz) [reply ipn/ipnlocal/local.go]: Rather, I moved the method there for now. We can do the map later.

**Pull request merged**